docs/configuring-metabase/email.md

```diff
@@ -1,10 +1,10 @@
 ---
-title: Email
+title: Configure email
 redirect_from:
   - /docs/latest/administration-guide/02-setting-up-email
 ---
 
-# Email
+# Configure email
```

Comment:
We can leave it as "email" in the sidebar but I think for H1 (so stuff that comes up in search) it's better to be more descriptive